cmd/schedule-builder/cmd/model.go

```diff
@@ -38,3 +38,38 @@ type Schedule struct {
 	EndOfLifeDate      string            `yaml:"endOfLifeDate"`
 	PreviousPatches    []PreviousPatches `yaml:"previousPatches"`
 }
+
+type ReleaseSchedule struct {
+	Releases []Release `yaml:"releases"`
+}
+
+type Release struct {
+	Version      string        `yaml:"version"`
+	Links        []Link        `yaml:"links"`
+	TrackingDocs []TrackingDoc `yaml:"trackingDocs"`
+	Guides       []Guide       `yaml:"guides"`
```

Comment:
I think we do not have to need them configurable. If we use a go template, then changing the template seems sufficient to me.

Reply:
Thanks a lot for this suggestion, I think the links within []Link and TrackingDocs[] will change with every release? I haven't worked with go template earlier so I have little idea if we can manage it without making it configurable in our YAML template?

Reply:
Yes the links change, but we can inherit them mainly from the version, see:

```markdown
* [This document](https://git.k8s.io/sig-release/releases/release-1.22/README.md)
* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.22/release-team.md)
* [Meeting Minutes](http://bit.ly/k8s122-releasemtg) (join [kubernetes-sig-release@] to receive meeting invites)
* [v1.22 Release Calendar][k8s122-calendar]
* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
* [Internal Contact Info][Internal Contact Info] (accessible only to members of [kubernetes-release-team@])
```

Reply:
Thank you so much, this makes everything a lot more clear, I will fix this!